HIS-28: Add Ozone Logging Documentation

enyachoke opened the pull request:
This is the initial documentation for Ozone Logging. It is part of [HIS-28](https://openmrs.atlassian.net/browse/HIS-28). This does not represent the complete documentation, but provides enough documentation to get started, and puts in place a structure to be filled in as we gather more information while using the logging service.

[HIS-28]: https://openmrs.atlassian.net/browse/HIS-28?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

rbuisson commented:
As discussed, @enyachoke  could you strip out the creation of charts/dashboard part to keep only the explore part?

enyachoke commented:
@rbuisson  I have updated the docs removing the creation of charts/dashboards. I have also fixed the rate queries that were having cardinality issues.
